includes/controller/user_settings_controller.php

```diff
@@ -187,12 +188,48 @@ function user_settings() {
         $display_message=strip_request_item('display_message');
       else
         $ok = false;
-      if($ok){
+
+      if ($ok){
         update_display_msg($display_message);
         success("Message Changed");
         redirect(page_link_to('user_settings'));
       }
   }
+  elseif (isset($_REQUEST['update'])) {
+    $ok = true;
+
+    $online_ver = file_get_contents("https://raw.githubusercontent.com/fossasia/engelsystem/master/Version.txt");
+    $current_ver = file_get_contents(" ./Version.txt");
```

Comment:
All the links are wrong. According to the work flow we include all the files in public/index.php. We need to give path relative to public folder.
Please refer this code https://github.com/fossasia/engelsystem/blob/development/includes/model/import_model.php.